### DISCUSSION TITLE
We did it folks.

### DISCUSSION THREAD

**Discussion by adamwathan** [tailwindlabs/tailwindcss#1422] [General]
Discussions! I don’t know what I want to say but had to be first.

Stoked to see where things go with this!

**Comment by ericlbarnes** [comment]
So flipping exciting! 🥳🤩🥳😍

**Comment by dixonge** [comment]
just curious - are you an early invitee? IMHO *every* repo needs this turned on ASAP...

**Comment by adamwathan** [comment]
Yeah part of the beta group 👍
